app/models/manageiq/providers/azure/cloud_manager.rb

```diff
@@ -80,6 +80,12 @@ def vm_start(vm, _options = {})
   end
 
   def vm_stop(vm, _options = {})
+    vm.deallocate
+  rescue => err
+    _log.error "vm=[#{vm.name}], error: #{err}"
+  end
+
+  def vm_suspend(vm, _options = {})
```

Comment:
As as above, here it calls `vm.suspend` and it eventually calls `vm.raw_suspend`